CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 - Fix: [#2691] Scenarios with original procedural terrain generation could crash.
 - Fix: [#2692] Inability to place certain track pieces.
 - Fix: [#2717] Crash when generating a landscape with the river meander rate set to 1.
+- Fix: [#2721] North arrow image not changing to reflect rotation on the Map window.
```

Comment:
2718 is the issue 2721 is the PR. We normally reference the issue here unless there isn't an issue in which case we would reference the PR.
Please change to 2718